src/styles/utils/_mixins.scss

```diff
@@ -1,4 +1,4 @@
-@import 'utils/colors';
+@import './colors';
```

Comment:
I don't think you need this import at all since `_colors` is imported after `_mixins` in main.scss.

Reply:
Ok I adjusted the path to avoid the compiling errors, but if we don't need them, it will be just better to remove them!! Thanks.